### DISCUSSION TITLE
player: add scrolling

### DISCUSSION THREAD

**Discussion by jbohren** [asciinema/discussions#147] [Ideas]
It would be great if when the user used the scroll wheel on the web viewer, it scrolled up as if it were a real terminal window. This would be great for looking over output that might get printed really quickly.

(migrated from https://github.com/asciinema/asciinema/issues/137)
@wjwwood

**Comment by SonGokussj4** [comment]
Hi. I've just found out about Asciinema and it's awesome. But right the first day this is a feature that's missing for me. Often there is much output and if viewer can't see the scrolling (either by mouse or by SHIFT+UP/DOWN or SHIFT+PGUP/PGDOWN, it's less usable.

**Comment by janluke** [comment]
First time user here. This was my first and only issue with asciicinema, but a severe one. As a workaround, one can pipe the program output to `less` or `more` but that's annoying and, at least on Windows (I'm using PowerSession), it removes colors.

**Comment by pronebird** [comment]
I am surprised this has not been implemented yet.

**Comment by ku1ik** [comment]
asciinema's virtual terminal emulator ([avt](https://github.com/asciinema/avt)) didn't support scrollback in the past, which was the blocker. Now the scrollback is there (since last year) it would be possible to have scrollback/scrolling implemented in the player. No promises though, maybe some day.
